src/main/java/org/jabref/gui/fieldeditors/contextmenu/ClearField.java

```diff
@@ -0,0 +1,15 @@
+package org.jabref.gui.fieldeditors.contextmenu;
+
+import javafx.scene.control.MenuItem;
+import javafx.scene.control.TextArea;
+
+import org.jabref.logic.l10n.Localization;
+
+class ClearField extends MenuItem {
+
+    public ClearField(TextArea opener) {
+        super(Localization.lang("Clear text"));
```

Comment:
You can also just reuse the text "Clear" here so that our poor translators don't have to work too much.